packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -292,12 +292,11 @@ export class Popover
     this.setFilteredPlacements();
     connectLocalized(this);
     connectMessages(this);
-    this.setUpReferenceElement(this.hasLoaded);
     connectFocusTrap(this);
 
-    if (this.open) {
-      onToggleOpenCloseComponent(this);
-    }
+    // we set up the ref element in the next frame to ensure PopoverManager
+    // event handlers are invoked after connect (mainly for `components` output target)
+    requestAnimationFrame(() => this.setUpReferenceElement(this.hasLoaded));
```

Comment:
Would we need to do this for the other `floating-ui` components? Or is it only popover because of the PopoverManager?

Reply:
I believe it's the latter. I tested with other components, but `popover` seemed to be the the only one affected. I'll do another sweep to make sure.

Reply:
Confirmed.